### PR TITLE
Files ready for static Review

### PR DESCRIPTION
**Pull request by Millsy66**
Static Review of Paul's Files using  the  following Style Guide:

•	All class, variable, and method names should be meaningful. E.g. for a method that lists books, the name should be listBooks(), rather than lbk(). 
•	Class, variable, and method names should not use abbreviations, except for common acronyms (e.g. html rather than HyperText Markup Language) 
•	Class names should start with a capital and use CamelBack. 
•	Variable names should be nouns or noun phrases, start with a lower case letter and use camelBack. 
•	Method names should be verbs or verb phrases, start with a lower case letter and useCamelBack. 
•	Indentation increments should be 4 characters. 
•	All variables and operators MUST be separated by white space. E.g.: 
Not: 
a=b+c+d; 
Rather: 
a = b + c + d; 
•	Anywhere curly brackets MAY be used, they MUST be used. E.g.: 
Not: 
if (condition) 
some single line of optional code; 
further non optional code; 
Rather: 
if (condition) { 
some single line of optional code; 
} 

•	Random rules that must also be enforced: 
•	There should be no method calls within argument lists of any method calls. 
•	No calculations within Boolean expressions 
•	No negated Boolean variable names or negated Boolean methods. E.g. ‘isError’ rather than ‘isNoError’. 
•	Singletons should return their sole instance through a method called getInstance() 
•	All class and method declarations should be separated from the previous block of code by 2 space lines.

### CODE REVIEW

File: ITC205_BaseLine_Code/src/FixBookControl.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 public class FixBookControl {
 	
+  //File ready for Static review
```
Comment (bcurby) [inline ITC205_BaseLine_Code/src/FixBookControl.java]: D1 - whole FixBookControl.java file - Change tabs -> 4 spaces

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D2 - line 24 FixBookControl.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D3 - line 44 FixBookControl.java - remove method call from argument list of ui.display()

Replace with:

String currentBookInfo = currentBook.toString();
	   ui.display(currentBookInfo);

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D4 - line 45 FixBookControl.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D5 - line 59 FixBookControl.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D6 - line 69 FixBookControl.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D7 - line 71 & 72 FixBookControl.java - Remove lines of whitespace between closing braces

File: ITC205_BaseLine_Code/src/FixBookUI.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import java.util.Scanner;
 
-
+//File ready for Static review
 public class FixBookUI {
 
 	public static enum UIState { INITIALISED, READY, FIXING, COMPLETED };
```
Comment (bcurby) [inline ITC205_BaseLine_Code/src/FixBookUI.java]: D8 - line 6 FixBookUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D9 - line 10 FixBookUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D10 - line 16 FixBookUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D11 - line 21 FixBookUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D12 - line 26 FixBookUI.java - Change method run -> runFixBook

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D13 - line 27 FixBookUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D14 - line 34 FixBookUI.java - Change method input -> displayInputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D15 - line 37 & 38 FixBookUI.java - Move else up onto line with closing brace on line 37 and then delete line 38 whitespace e.g. 
} else {

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D61 - line 41 FixBookUI.java - Change e -> exception

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D16 - line 42 FixBookUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D17 - line 48 FixBookUI.java - Change method input -> displayInputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D18 - line 59 FixBookUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D19 - line 63 FixBookUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D20 - line 70 FixBookUI.java - Change method input -> displayInputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D21 - line 76 FixBookUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D22 - line 81 FixBookUI.java - Change method display -> displayMessage

File: ITC205_BaseLine_Code/src/Loan.java [hunk 3]
```diff
@@ -2,6 +2,7 @@
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+//File ready for Static review
 @SuppressWarnings("serial")
 public class Loan implements Serializable {
```
Comment (bcurby) [inline ITC205_BaseLine_Code/src/Loan.java]: D23 - line 11 Loan.java - Change loanID -> loanId

Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D24 - line 20 Loan.java - Change loanID -> loanId

Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D25 - line 41 Loan.java - Change method getId -> getLoanId

Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D26 - line 42 Loan.java - Change loanID -> loanId

Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D27 - line 55 Loan.java - Change loanID -> loanId

Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D28 - line 58 Loan.java - Change (book.id()) -> (book.getBookId())

Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D29 - line 79 Loan.java - Remove line of whitespace between closing braces

File: ITC205_BaseLine_Code/src/PayFineControl.java [hunk 4]
```diff
@@ -1,5 +1,6 @@
 public class PayFineControl {
 	
+  //File ready for Static review
```
Comment (bcurby) [inline ITC205_BaseLine_Code/src/PayFineControl.java]: D30 - line 24 PayFineControl.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D31 - line 40 PayFineControl.java - remove method call from argument list of display
e.g.

ui.display(member.toString());

Replace with:

String memberInfo = member.toString();
	   ui.display(memberInfo);

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D32 - line 46 PayFineControl.java - Change method cancel -> cancelPayingFine

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D33 - line 47 PayFineControl.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D34 - line 59 PayFineControl.java - Change method display -> displayMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D35 - line 61 PayFineControl.java - remove method call from argument list of display
e.g.

ui.display(member.toString());

Replace with:

String memberInfo = member.toString();
	   ui.display(memberInfo);

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D36 - line 61 PayFineControl.java - Change method display -> displayMessage
e.g.

String memberInfo = member.toString();
ui.displayMessage(memberInfo);

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D37 - line 62 PayFineControl.java - Change UIState -> UiState

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import java.util.Scanner;
 
-
+//File ready for Static review
 public class PayFineUI {
 
 	public static enum UIState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
```
Comment (bcurby) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: D38 - line 6 PayFineUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D39 - line 10 PayFineUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D40 - line 16 PayFineUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D41 - line 21 PayFineUI.java - Change UIState -> UiState

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D42 - line 26 PayFineUI.java - Change method run -> runPayFine

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D43 - line 27 PayFineUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D44 - line 34 PayFineUI.java - Change method input -> displayInputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D45 - line 43 PayFineUI.java - Change e -> exception

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D46 - line 44 PayFineUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D47 - line 50 PayFineUI.java - Change method input -> displayInputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D48 - line 58 PayFineUI.java - Change e -> exception

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D49 - line 44 PayFineUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D50 - line 67 PayFineUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D51 - line 71 PayFineUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D52 - line 75 PayFineUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D53 - line 83 PayFineUI.java - Change method input -> displayInputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D54 - line 89 PayFineUI.java - Change method output -> displayOutputMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D55 - line 94 PayFineUI.java - Change method display -> displayMessage

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D56 - line 95 PayFineUI.java - Change method output -> displayOutputMessage

File: ITC205_BaseLine_Code/src/FixBookUI.java (hunk 2 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D57 - whole FixBookUI.java file - Change tabs -> 4 spaces

File: ITC205_BaseLine_Code/src/Loan.java (hunk 3 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D58 - whole Loan.java file - Change tabs -> 4 spaces

File: ITC205_BaseLine_Code/src/PayFineControl.java (hunk 4 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D59 - whole PayFineControl.java file - Change tabs -> 4 spaces

File: ITC205_BaseLine_Code/src/PayFineUI.java (hunk 5 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D60 - whole PayFineUI.java file - Change tabs -> 4 spaces

**Review by bcurby — CHANGES_REQUESTED** [review]
Please fix D1 - D61 for static review

File: ITC205_BaseLine_Code/src/Loan.java (hunk 3 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D62 - line 58 Loan.java - Change book.title -> book.getTitle 

Method 'title' is in Book.java on line 44

File: ITC205_BaseLine_Code/src/FixBookControl.java (hunk 1 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D63 - line 12 FixBookControl.java - Add extra line of whitespace above public FixBookControl() so there are 2 lines of whitespace between it and previous block of code

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D64 - line 4-6 FixBookControl.java - Change private enum class to:

    private enum ControlState { INITIALISED, READY, FIXING };

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookControl.java]: D65 - line 21, 31, 52 & 65 FixBookControl.java - Apply line wrapping so lines don't exceed 100 character limit e.g.

            throw new RuntimeException("FixBookControl:"
                + " cannot call bookScanned except in READY state");

File: ITC205_BaseLine_Code/src/FixBookUI.java (hunk 2 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D66 - line 6 - 8 FixBookUI.java - Change enum class to:

    public static enum UiState { INITIALISED, READY, FIXING, COMPLETED };

Also only one line of whitespace between enum and variables below

Comment (bcurby) [reply ITC205_BaseLine_Code/src/FixBookUI.java]: D67 - line 15 FixBookUI.java - Add line of whitespace above FixBookUI declaration so there is the required 2 lines of space

File: ITC205_BaseLine_Code/src/Loan.java (hunk 3 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D68 - line 9-11 Loan.java - Change enum class to:

    public static enum LoanState { CURRENT, OVER_DUE, DISCHARGED };

Also only one line of whitespace between enum and variables below

Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D69 - line 20 Loan.java - Add line of whitespace above method declaration so there are 2 lines of whitespace

File: ITC205_BaseLine_Code/src/PayFineControl.java (hunk 4 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D70 - line 4-6 PayFineControl.java - Change enum class to:

    private enum ControlState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };

Also only one line of whitespace between enum and variables below

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D71 - line 13 PayFineControl.java - Add line of whitespace above PayFineControl() declaration so there are 2 lines of whitespace

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineControl.java]: D72 - line 21, 31 & 54 PayFineControl.java - Apply line wrapping so lines don't exceed limit of 100 characters e.g.

            throw new RuntimeException("PayFineControl:"
                + " cannot call payFine except in PAYING state");

Double check indent for new line is 4 spaces

File: ITC205_BaseLine_Code/src/PayFineUI.java (hunk 5 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D73 - line 6-8 PayFineUI.java file - Change enum class to:

    public static enum UiState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };

Also only one line of whitespace between enum and variables below

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D74 - line 15 PayFineUI.java - Add line of whitespace above PayFineUI() declaration so there are 2 lines of whitespace

Comment (bcurby) [reply ITC205_BaseLine_Code/src/PayFineUI.java]: D75 - line 78 PayFineUI.java - Remove line of whitespace between code and closing braces

**Comment by bcurby** [comment]
Please also fix D62 - 75

File: ITC205_BaseLine_Code/src/Loan.java (hunk 3 above)
Comment (bcurby) [reply ITC205_BaseLine_Code/src/Loan.java]: D76 - line 56-59 Loan.java - Apply line-wrapping so these lines don't exceed the 100 character limit. Also apply 4 space indentation e.g.
Before:
before

After:
after

**Comment by bcurby** [comment]
Please also fix D76

**Comment by bcurby** [comment]
All defects (D1 - D76) checked and requested changes confirmed.
Changes will now be approved.
Ready for Facilitator check and merge.

**Review by bcurby — APPROVED** [review]

**Comment by lukexjohnson** [comment]
LUKEJ "mediator/facilitator": I have reviewed the code. Defects have been completed and align correctly, program runs well in IDE. Happy to proceed with merge.

**Pull request merged**